dashboard/django/collector.py

```diff
@@ -355,6 +355,8 @@ def fetch_stats():
         if cluster_name == 'global': continue
 
         cluster_url = get(args.serviceUrl, '/admin/v2/clusters/' + cluster_name)['serviceUrl']
+        if cluster_url.find(',')>=0:
+                cluster_url = cluster_url.split(',')[0]
```

Comment:
with this change, the collector will only fetch stats from one broker. not sure if there is a better approach here. for example, can we keep a list of servers and fail over better servers?

Reply:
import random

Reply:
How about using random?